packages/storage/src/chrome/test-utils/tests-setup.js

```diff
@@ -23,3 +23,8 @@ globalThis.chrome = {
 
 globalThis.DEFAULT_GRPC_URL = 'https://rpc.example.com/';
 globalThis.MINIFRONT_URL = 'https://app.example.com';
+globalThis.process = {
+  env: {
+    NODE_ENV: 'test',
```

Comment:
The new `IndexedDb#iterateAuctions()` method calls `base64ToUint8Array()`, which calls `validateSchema()`, which calls `isDevEnv()`, which calls `getEnv()`, which expects `process.env.NODE_ENV` to exist :)